modules/project-docs/pages/compatibility.adoc

```diff
@@ -93,9 +102,9 @@ The SDK follows http://semver.org/[Semantic Versioning^] and typically release a
 In order to update your specific developer environment to this new version, you will have to instruct Go to `go get` that specific version.
 For further information, please reference the section on installation.
 
-include::6.5@sdk:shared:partial$network-requirements.adoc[]
+include::6.6@sdk:shared:partial$network-requirements.adoc[]
 
-NOTE: With Go 1.6, which does not have the Alternate Address feature, you need to use `?network=default` in the connection string for COuchbase Cloud.
+NOTE: With Go 1.6, which does not have the Alternate Address feature, you need to use `?network=default` in the connection string for Couchbase Cloud.
```

Comment:
This isn't really related to this PR I guess but 1.6 *does* have the alternate address feature and doesn't require a query string param - even if it did it would need network=external anyway :D.